docs/common_issues.md

```diff
@@ -0,0 +1,140 @@
+# Common issues during build and check tasks
+
+This page highlights some common issues and fixes.
+
+> [!TIP]
+> Doing the check is recommended before and after making any
+> change to a schema when having issues.
+
+## Common solution
+
+The common solution is:
+
+1. Edit current.json file to fix problem as highlighted below.
+
+2. Target the build: `npm run build-new path-to-current-dot-json`
+   or `npm run build-modified` if changes were made to a
+   committed files. The `npm run build-all` command can
+   make it more difficult to track down some issues
+   and fix them.
+
+3. Do check via: `npm run check` 
+
+Issues prefixed with "check" occur during step 3 and others
+during step 2.
+
+> [!TIP]
+> Create schema that do not use $ref keyword first. When 2 and 3
+> complete create a new schema where the $ref is the $id you
+> just checked is okay and build will merge everything
+> into a single file.
+
+### Using values in $id that never change (ie. draft schema)
+
+The value in $id is important and **must be** correct. A "draft"
+schema can retain the same value (where $id ending
+with "/0.0.1" is recommended at start) and builds overwrites.
+An $id ending in 1.0.0 or higher
+is considered "completed" where the $id should be alter
+with certain patterns to increase robustness of schema
+change over time so all old, current and future
+versions are available at the same time.
+
```

Comment:
```suggestion
```

I think this bit is a bit opinionated and we should allow people a bit of freedom in how they approach things. I would suggest we leave this out for now. I don't think this really is a common issue either.